app/web/features/profile/view/About.test.tsx

```diff
@@ -36,6 +36,57 @@ describe("About (user)", () => {
     ).toBe(1);
   });
 
+  it("should display age and gender without verification icons when unspecified ", async () => {
+    renderAbout({
+      ...defaultUser,
+      genderVerificationStatus: 0,
+      birthdateVerificationStatus: 0,
+    });
+
+    const checkIcon = screen.queryByTestId("check-circle-icon");
+    expect(checkIcon).toBeNull();
+    const errorIcon = screen.queryByTestId("error-icon");
+    expect(errorIcon).toBeNull();
+  });
+  it("should display age and gender without verification icons when unverified", async () => {
+    renderAbout({
+      ...defaultUser,
+      genderVerificationStatus: 1,
```

Comment:
can we use strings as values instead of plain integers? (just to make it more readable :) )

Reply:
genderVerificationStatus: "unverified" | "verified" | "otherstate"

Reply:
same for birthdateVerificationStatus :)

Reply:
This is an api change but i agree with this as well. It was a bit confusing for me to understand what the numbers meant until i went through the typings. I see we use this number system for  most of the other keys as well.

Reply:
They are enums. They should be exported in the generated protobuf code so you can use those instead of numbers in here, but I guess they map to ints in the end.